recipes/conversation_view/feed_pill/feed_item_pill.vue

```diff
@@ -0,0 +1,147 @@
+          :aria-label="ariaLabel"
+          data-qa="dt-feed-item-pill"
+          :class="['d-baw0 d-bgc-moderate d-bar-pill d-w100p d-ta-left', buttonClass]"
+          @focusin="hover = true"
+          @focusout="hover = false"
+          @mouseenter="hover = true"
+          @mouseleave="hover = false"
+          @click="$emit('click')"
+        >
+          <dt-item-layout class="d-w100p d-p8">
+            {{ title }}
+            <template slot="left">
+              <dt-icon
+                v-if="toggleable && hover"
+                class="dt-feed-icon"
+                data-qa="dt-feed-item-hover-icon"
+                :name="expanded ? 'chevron-down' : 'chevron-right'"
+              />
+              <dt-icon
+                v-else
+                class="dt-feed-icon"
+                data-qa="dt-feed-item-icon"
+                :name="iconName"
+                size="300"
+              />
+            </template>
+            <template slot="subtitle">
+              <slot name="subtitle" />
+            </template>
+            <template slot="bottom">
+              <slot name="bottom" />
+            </template>
+            <template slot="right">
+              <slot name="right" />
+            </template>
+          </dt-item-layout>
+        </button>
+        <div
+          v-if="expanded"
+          class="d-jc-center d-d-flex"
+        >
+          <slot name="content" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { DtIcon, DtItemLayout } from '@/index';
+
+export default {
+  name: 'DtRecipeFeedItemPill',
+
+  components: { DtItemLayout, DtIcon },
+
+  mixins: [],
+
+  props: {
+    /**
+     * Accepts a DtIcon name to be shown in the left
+     */
+    iconName: {
+      type: String,
+      default: () => '',
+    },
+
+    /**
+     * Bolded primary text
+     */
+    title: {
+      type: String,
+      default: () => '',
+    },
+
+    /**
+     * Shows/hides the expanded portion
+     */
+    expanded: {
+      type: Boolean,
+      default: () => false,
+    },
+
+    /**
+     * Additional styling around the pill
+     */
+    wrapperClass: {
+      type: String,
+      default: () => '',
+    },
+
+    /**
+     * Additional styling for the pill
+     */
+    buttonClass: {
+      type: String,
+      default: () => '',
+    },
+
+    /**
+     * Aria label for feed pill
+     */
+    ariaLabel: {
+      type: String,
+      default: () => '',
+    },
+
+    /**
+     * Sets whether the pill can be toggled
+     */
+    toggleable: {
+      type: Boolean,
+      default: () => true,
+    },
+  },
+
+  emits: [
+    'click',
+  ],
+
+  data () {
+    return {
+      hover: false,
+    };
+  },
+};
+</script>
+
+<style lang="less">
+  .dt-feed-item--expanded-pill {
+    border-radius: 5rem
+  }
+
+  .dt-feed-icon {
+    animation: fade 0.2s ease-in;
+  }
+
+  @keyframes fade {
+    0%   {transform: scale(0);}
+    100% {transform: scale(1);}
+  }
+</style>
```

Comment:
Styles for recipe components need to be scoped https://github.com/dialpad/dialtone-vue/blob/staging/.github/CONTRIBUTING.md#coding-guidelines

Also make sure to be using [BEM naming convention](https://getbem.com/introduction/)